src/util.js

```diff
@@ -73,7 +73,7 @@ function parseContextFromHeader(str) {
   return {
     traceId: matches[1],
     spanId: matches[2],
-    options: Number(matches[3])
+    options: typeof(matches[3]) === 'undefined' ? 0 : Number(matches[3])
```

Comment:
Also, maybe `Number(matches[3] || 0)` is more readable? In either case, could you add a comment as well?